vassal-app/src/main/java/VASSAL/configure/TranslatingStringEnumConfigurer.java

```diff
@@ -51,6 +52,24 @@ public TranslatingStringEnumConfigurer(String key, String name, String[] validVa
     this.i18nKeys = i18nKeys;
   }
 
+  /**
+   * Create a drop-list of localised display values that allows you to select a value from an
+   * underlying list of 'internal' untranslated values.
+   *
+   * @param key Configurer Key
+   * @param name Configurer Name
+   * @param validValues Array of values to maintain
+   * @param i18nKeys Array of Translation Keys used to describe the list of values
+   * @param isDisplayNames True if we have already been given full translated strings; false if they are "keys" for Resources.getString()
+   */
```

Comment:
Change the old Constructors without the isDisplayName parameter to either call the new constructor with isDisplayName=false or if not possible just set this.isDisplayName = false,